README.md

```diff
@@ -88,23 +88,25 @@ You can initialize the FSRS scheduler with your own custom weights as well as de
 ```python
 f = FSRS(
     w=(
-        1.14,
-        1.01,
-        5.44,
-        14.67,
-        5.3024,
-        1.5662,
-        1.2503,
-        0.0028,
-        1.5489,
-        0.1763,
-        0.9953,
-        2.7473,
-        0.0179,
-        0.3105,
-        0.3976,
-        0.0,
-        2.0902,
+        0.4197,
+        1.1869,
+        3.0412,
+        15.2441,
+        7.1434,
+        0.6477,
+        1.0007,
+        0.0674,
+        1.6597,
+        0.1712,
+        1.1178,
+        2.0225,
+        0.0904,
+        0.3025,
+        2.1214,
+        0.2498,
+        2.9466,
+        0.4891,
+        0.6468,
```

Comment:
I believe the default parameters should be consistent with the documentation.
- https://github.com/open-spaced-repetition/fsrs4anki/wiki/The-Algorithm#default-parameters
- https://github.com/open-spaced-repetition/fsrs-rs/blob/840d80e8b017e78af0f7484f11d500adc5a06388/src/inference.rs#L24-L27

Reply:
Yeah that works too. I initially was thinking of specifying different parameters in the README example just to show clearly that you can specify parameters different than the defaults, but this is fine 👍